rest/data/interface.go

```diff
@@ -90,6 +90,12 @@ type Connector interface {
 	CreateProject(projectRef *model.ProjectRef) error
 	UpdateProject(projectRef *model.ProjectRef) error
 
+	// EnableWebhooks creates a webhook for the project's owner/repo if one does not exist.
+	// If unable to setup the new webhook, returns false but no error.
+	EnableWebhooks(*model.ProjectRef) (bool, error)
```

Comment:
can this just take projectIDs? do we need the refs?

Reply:
We need repo, owner, and projectID, so I think passing the ref is cleaner than passing 3 strings separately.